.gitmodules

```diff
@@ -37,3 +37,6 @@
 [submodule "sublime_syntaxes/Swift-for-f-ing-sublime"]
 	path = sublime_syntaxes/Swift-for-f-ing-sublime
 	url = https://github.com/colinta/Swift-for-f-ing-sublime.git
+[submodule "sublime_syntaxes/PowerShell-EditorSyntax"]
```

Comment:
I think we don't need the submodules since you have converted the `tmLanguage` into `sublime-syntax` and we do end up not using anything from this repo?

Reply:
Maybe not necessary, but I thought I was following examples, e.g. the submodule `Sublime-VimL` only has a `tmLanguage` file.

Reply:
Ah missed that one, I will need to check which submodules are needed